src/engine-actions/get-chain-transactions.js

```diff
@@ -0,0 +1,194 @@
+const {
+  getTransactions,
+  listClosedChannels,
+  listPendingChannels,
+  listChannels
+} = require('../lnd-actions')
+const { Big } = require('../utils')
+
+/**
+ * @constant
+ * @type {Object}
+ * @default
+ */
+const TRANSACTION_TYPES = Object.freeze({
+  CHANNEL_OPEN: 'CHANNEL_OPEN',
+  CHANNEL_CLOSE: 'CHANNEL_CLOSE',
+  DEPOSIT: 'DEPOSIT',
+  WITHDRAW: 'WITHDRAW',
+  PENDING: 'PENDING',
```

Comment:
Is `PENDING` really a transaction type? Seems more like a status of a transaction which has some other type (e.g. a pending channel open transaction vs a confirmed channel open transaction vs a pending channel close transaction)

Reply:
@treygriffith I can try to be more specific, but used PENDING just because I didn't want to add all of the specific types yet. PENDING is basically "waiting to be put into a block"

Reply:
Honestly, its probably worth it to be specific, let me see what we can do